examples/phase_change/ex2/input2d

```diff
@@ -465,6 +468,14 @@ RelaxationLSMethod {
     apply_volume_shift = APPLY_VOLUME_SHIFT
 }
 
+LevelSetMassFixer {
+   half_width = NUMBER_OF_INTERFACE_CELLS  // width of the interface in each phase to compute Heaviside integrals
+   max_its = 4                 // maximum no. of Newton's iterations
+   rel_tol = 1e-12             // error between current and target volume
+   correction_interval = 1     // frequency at which mass loss is corrected
+   enable_logging      = TRUE  // prints Netwon residuals
+}
+
```

Comment:
Are we using this functionality in phase change examples?

Reply:
I just added the mass loss fix to this example. This is a advection of bubble case. There is no phase change. So I added the mass loss fix to this example. But I am not using mass loss fix here.

Reply:
I am also adding three more examples: Bubble trap, Stefan melting (two-phase and three-phase). I am working on that. I will also test the restart and push it to this branch. Then this branch is ready for review.